### DISCUSSION TITLE
cannot see roles & permission in strapi panel

### DISCUSSION THREAD

**Discussion by nhd2106** [strapi/strapi#7938] [Legacy Topics]
i've just tried to create my very first strapi app with both yarn and npx.
but i could not see roles & permissions plugin in the panel.
have you guys ever got this issues?
many thanks.

**Comment by MattieBelt** [comment]
Hey @nhd2106,

Yesterday in `v3.1.5` the Roles & Permissions plugin has been moved to settings.
![image (3)](https://user-images.githubusercontent.com/31662805/93497055-47588d00-f910-11ea-8fca-3c469c3654ea.png)

#7467
